api/controllers/items.js

```diff
@@ -0,0 +1,70 @@
+const express = require('express');
+const router = express.Router();
+const db = require('../models');
+const { Item } = db;
+
+//    GET    /items
+//    POST   /items
+//    GET    /items/:id
+//    PUT    /items/:id
+//    DELETE /items/:id
+
+router.get('/', (req, res) => {
+  Item.findAll({}).then((items) => res.json(items));
+});
+
+router.post('/', (req, res) => {
+  let { content } = req.body;
+
+  Item.create({ content })
+    .then((item) => {
+      res.status(201).json(item);
+    })
+    .catch((err) => {
+      res.status(400).json(err);
+    });
+});
+
+router.get('/:id', (req, res) => {
+  const { id } = req.params;
+
+  Item.findByPk(id).then((item) => {
+    if (!item) {
+      return res.sendStatus(404);
+    }
+    res.json(item);
+  });
+});
+
+router.put('/:id', (req, res) => {
```

Comment:
If item exist, then update if data change.
If item does not exist, then create a new row.
so this might need to be change i think:
if (!item) {
      return res.sendStatus(404);
    }

Reply:
Good catch; implemented this.